WooCommerce/Classes/ViewRelated/Point Of Sale/PointOfSaleEntryPoint.swift

```diff
@@ -0,0 +1,25 @@
+import SwiftUI
+
+struct PointOfSaleEntryPoint: View {
```

Comment:
nit: my bad for not having updated the naming convention doc p1655276782116439/1651641104.110959-slack-CGPNUU63E, we've been naming views with the `View` suffix unless it's an obvious view name. An "entry point" sounds like not necessarily a view, I'd suggest adding the `View` suffix for clarity.

(Similar for the dashboard view, but up to you.)

```suggestion
struct PointOfSaleEntryPointView: View {
```

Reply:
Makes sense! Updated: https://github.com/woocommerce/woocommerce-ios/pull/12653/commits/94ce669b8df1bea71c61818459f39c9db5fb616d